pyfar/constants/air_attenuation.py

```diff
@@ -0,0 +1,243 @@
+            - frequency-to-pressure ratio: 4 x 10-4 Hz/Pa to 10 Hz/Pa.
+
+        ``50``, +/- 50% accuracy
+            - molar concentration of water vapour: less than 0,005%
+            - air temperature: greater than 200 K (- 73 °C)
+            - atmospheric pressure: less than 200 kPa (2 atm)
+            - frequency-to-pressure ratio: 4 x 10-4 Hz/Pa to 10 Hz/Pa.
+
+        ``-1``, no valid result
+            else.
+
+    References
+    ----------
+    .. [#] ISO 9613-1:1993, Acoustics -- Attenuation of sound during
+           propagation outdoors -- Part 1: Calculation of the absorption of
+           sound by the atmosphere.
+    """
+    # check inputs
+    if not isinstance(temperature, (int, float, np.ndarray, list, tuple)):
+        raise TypeError(
+            'temperature must be a number or array of numbers')
+    if not isinstance(frequency, (int, float, np.ndarray, list, tuple)):
+        raise TypeError(
+            'frequency must be a number or array of numbers')
+    if not isinstance(
+            relative_humidity, (int, float, np.ndarray, list, tuple)):
+        raise TypeError(
+            'relative_humidity must be a number or array of numbers')
+    if not isinstance(
+            atmospheric_pressure, (int, float, np.ndarray, list, tuple)):
+        raise TypeError(
+            'atmospheric_pressure must be a number or array of numbers')
+    if not isinstance(calculate_accuracy, bool):
+        raise TypeError(
+            'calculate_accuracy must be a bool.')
+
+    # check limits
+    if np.any(np.array(temperature) < -73):
+        raise ValueError("Temperature must be between -73°C.")
+    if np.any(np.array(frequency) < 50):
+        raise ValueError("Frequency must be greater than 50 Hz.")
+    if np.any(np.array(relative_humidity) < .1) or np.any(
+            np.array(relative_humidity) > 1):
+        raise ValueError("Relative humidity must be between 0.1 and 1.")
+    if np.any(np.array(atmospheric_pressure) > 200000):
+        raise ValueError("Atmospheric pressure must less than 200 kPa.")
+
+    # convert arrays
+    all_are_singles = isinstance(temperature, (float, int))
+    all_are_singles = all_are_singles and isinstance(frequency, (float, int))
+    all_are_singles = all_are_singles and isinstance(
+        relative_humidity, (float, int))
+    if not all_are_singles:
+        shape = np.broadcast_shapes(
+            np.array(temperature).shape,
+            np.array(frequency).shape,
+            np.array(relative_humidity).shape,
+            np.array(atmospheric_pressure).shape)
+        temperature = np.broadcast_to(
+            np.array(temperature, dtype=float), shape)
+        frequency = np.broadcast_to(
+            np.array(frequency, dtype=float), shape)
+        relative_humidity = np.broadcast_to(
+            np.array(relative_humidity, dtype=float), shape)
+        atmospheric_pressure = np.broadcast_to(
+            np.array(atmospheric_pressure, dtype=float), shape)
```

Comment:
Is the explicit broadcasting needed? I would suggest to convert input to numpy arrays in any case and maybe only
```
try:
    _ = np.broadcast_shapes(...)
except: ValueError as e:
    raise ValueError('some message') from e
```